src/analyses/interval_analysis.cpp

```diff
@@ -7,14 +7,25 @@ Author: Daniel Kroening, kroening@kroening.com
 \*******************************************************************/
 
 /// \file
-/// Interval Analysis
+/// Interval Analysis -- implements the functionality necessary for performing
+/// abstract interpretation over interval domain for goto programs. The result
+/// of the analysis is an instrumented program.
 
 #include "interval_analysis.h"
 
 #include <util/find_symbols.h>
 
 #include "interval_domain.h"
 
+/// Instruments all "post-branch" instructions with assertions about variable
```

Comment:
I guess technically we are instrumenting with `assumptions` and not `assertions` ?